wsd/DocumentBroker.cpp

```diff
@@ -1752,7 +1753,7 @@ void DocumentBroker::handleUploadToStorageResponse(const StorageBase::UploadResu
                                                    << filenameAnonym << "] successfully.");
 
                 std::ostringstream oss;
-                oss << "saveas: url=" << url << " filename=" << encodedName
+                oss << (_uploadRequest->isExport() ? "exportas:" : "saveas:") << " url=" << url << " filename=" << encodedName
```

Comment:
Why is it important to differentiate between the exportas URL and the saveas URL in the client? 

Do we even need to tell the client the URL of the export?

Reply:
we need to return the file name only as it can differ from what user typed (when file already exists in the storage)

Reply:
My question was whether the client code differentiates between an exportas and saveas. I assume it does, otherwise we could just send "saveas" at this line.